docs/devguide/quality_of_service.rst

```diff
@@ -120,1363 +103,1542 @@ The following examples illustrate how to obtain the default policies for publish
+
+.. _qos-ownership-strength:
+.. _quality_of_service--ownership-strength:
+
+Ownership Strength QoS
+======================
 
 ..
-    Sect<3.2.16>
+    Sect<3.2.23>
 
-The ``ENTITY_FACTORY`` policy controls whether entities are automatically enabled when they are created.
-Below is the IDL related to the Entity Factory QoS policy:
+The ownership strength QoS policy is used in conjunction with the :ref:`qos-ownership` ``EXCLUSIVE_OWNERSHIP_QOS`` policy on :term:`DataWriter`\s.
 
-.. code-block:: omg-idl
+.. important::
 
-    struct EntityFactoryQosPolicy {
-      boolean autoenable_created_entities;
+  This policy is :ref:`mutable <qos-changing>` and does not affect association.
+
+  IDL:
+
+  .. code-block:: omg-idl
+
+    struct OwnershipStrengthQosPolicy {
+      long value;
     };
 
-This policy can be applied to entities that serve as factories for other entities and controls whether or not entities created by those factories are automatically enabled upon creation.
-This policy can be applied to the domain participant factory (as a factory for domain participants), domain participant (as a factory for publishers, subscribers, and topics), publisher (as a factory for data writers), or subscriber (as a factory for data readers).
-The default value for the ``autoenable_created_entities`` member is ``true``, indicating that entities are automatically enabled when they are created.
-Applications that wish to explicitly enable entities some time after they are created should set the value of the ``autoenable_created_entities`` member of this policy to ``false`` and apply the policy to the appropriate factory entities.
-The application must then manually enable the entity by calling the entity's ``enable()`` operation.
+  .. list-table::
+     :header-rows: 1
 
-The value of this policy may be changed at any time.
-Changes to this policy affect only entities created after the change.
+     * - :term:`Applicable entities <Entity>`
 
-.. _quality_of_service--presentation:
+       - Members
+
+       - :ref:`Default values <qos-defaults>`
+
+     * - :term:`DataWriter`
+
+       - ``value``
 
-PRESENTATION
+       - ``0``
+
+  Specification: :omgspec:`dds:2.2.3.10 OWNERSHIP_STRENGTH`
+
+The ``value`` member is used to determine which data writer is the *owner* of the :term:`instance`.
+The default value is zero.
+
+.. _qos-property:
+
+Property QoS
 ============
 
-..
-    Sect<3.2.17>
+The property QoS policy contains sequences of key-value pairs for the :term:`DomainParticipant`.
 
-The ``PRESENTATION`` QoS policy controls how changes to instances by publishers are presented to data readers.
-It affects the relative ordering of these changes and the scope of this ordering.
-Additionally, this policy introduces the concept of coherent change sets.
-Here is the IDL for the Presentation QoS:
+.. important::
 
-.. code-block:: omg-idl
+  This policy is :ref:`immutable <qos-changing>` and affects association indirectly through security.
```

Comment:
I almost forgot about this, but I couldn't find a answer to if the DomainParitcipant properties QoS is actually immutable or not. I just assumed it was immutable, but it doesn't say either in the security spec or in the QoS code:

https://github.com/OpenDDS/OpenDDS/blob/33f65f988808d814f3828bf090ba0eab7c2b9c07/dds/DCPS/Qos_Helper.inl#L1129-L1135

I guess going by the code it's mutable if it's not checked.

Reply:
It's considered "Changeable" (using the term from the core spec) even though this isn't explicitly stated.
One way to see it is that it would be awkward for other specs to introduce additional constraints on the core's behavior.  The core spec offers no framework for extensibility of QoS by implementations or by other specs.

Reply:
Okay, so if the actual QoS values are changeable I assume this doesn't mean security will start using a different set of security documents that were passed in? Even if the specs aren't ideal about this, I'd like to document behavior the user might care about it.

Reply:
The propertyQos is changeable since you can use it for anything (or a combination of things).  There are some cases where it would make sense to change the security properties, but I think they're not currently implemented in OpenDDS.